licenses/LICENSE-REDIS

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2013 The github.com/go-redis/redis Authors.
```

Comment:
add license https://github.com/chenyahui/gin-cache

Reply:
i got it, thx